content/root/best-practice-guide.textile

```diff
@@ -117,8 +117,8 @@ h3(#choosing-between-realtime-rest-mqtt). Using the correct client library SDK -
 * If you just want to just publish data to the platform and don't need to subscribe or manage channel attachment, you can the "REST library":/rest.
 * If you want to publish at very high volumes or with the lowest possible realtime latencies, "Ably's realtime library would be best as it can publish at higher volumes and lower latencies than is possible with the REST library due to message pipelining":https://knowledge.ably.com/should-i-use-the-rest-or-realtime-library.
 * If you want to get one-time info about the presence set of a channel, you can use the "REST library":/rest.
-* If you want to set up realtime communication with IoT devices on platforms for which there's no native Ably client library available, you can the "MQTT protocol adapter":/mqtt.
-* If you wish to migrate from PubNub/Pusher to Ably, you can use the relevant "protocol adapter":/adapters to "easily":https://knowledge.ably.com/can-you-help-ease-the-pain-of-migration-from-another-data-stream-network switch to Ably right away and gradually migrate to using native Ably client SDKs.
+* If you want to set up realtime communication with IoT devices on platforms for which there's no available Ably client library, you can use the "MQTT protocol adapter":/mqtt.
+* If you wish to migrate from PubNub/Pusher to Ably, you can use the relevant "protocol adapter":/adapters to "easily":https://knowledge.ably.com/can-you-help-ease-the-pain-of-migration-from-another-data-stream-network switch to Ably right away and gradually migrate to using Ably client SDKs.
```

Comment:
I've reworded this and removed native in my PR. I suggest we merge your PR first, and then I can deal with the merge conflicts.